README.md

```diff
@@ -26,8 +25,25 @@ See .env.example
 Use the [release process template in Trello](https://trello.com/c/8enGdMyy) to
 start a new release.
 
+## Including Markdown in the workflow
```

Comment:
:heart: this! I would like to split this out into it's own file and link to it, but not now!